modules/seqexec/model/src/test/scala/seqexec/model/SeqexecModelArbitraries.scala

```diff
@@ -26,12 +25,6 @@ trait SeqexecModelArbitraries extends ArbObservation {
 
   implicit val clientIdArb: Arbitrary[ClientID] = Arbitrary(Gen.uuid)
 
-  implicit val instArb: Arbitrary[Instant] = Arbitrary {
-    for {
-      i <- Gen.choose(0L, Long.MaxValue)
-    } yield Instant.ofEpochMilli(i)
-  }
-
```

Comment:
Why are we getting rid of the arbitrary `Instant`?

Reply:
Because there is another one defined on the `core` project. I'm using that instead